src/com/google/enterprise/adaptor/DocumentHandler.java

```diff
@@ -185,6 +209,15 @@ private void initFullAccess(String gsaHostname, String[] fullAccessHosts) {
             + "access to content: {0}", new Object[] {addressesAndRanges});
   }
 
+  private boolean requestIsFromSkipCertAddresses(HttpExchange ex) {
```

Comment:
Hi Szabi,

Thank you for taking a look. We decided with Sergii to split off
the part where variable is integrated into DocumentHandler.
DocumentHandler is possibly the most complex class in the
library and I believe review will be simpler if this part is done
with before we integrate the logic.

Thank you


-
- technology's compounding interest

On Fri, Aug 26, 2016 at 1:23 AM, Szabolcs Fruhwald <notifications@github.com
> wrote:

> In src/com/google/enterprise/adaptor/DocumentHandler.java
> <https://github.com/googlegsa/library/pull/13#discussion_r76381813>:
>
> > @@ -185,6 +209,15 @@ private void initFullAccess(String gsaHostname, String[] fullAccessHosts) {
> >              + "access to content: {0}", new Object[] {addressesAndRanges});
> >    }
> >
> > +  private boolean requestIsFromSkipCertAddresses(HttpExchange ex) {
>
> I cannot see a single call to this method.. how this feature works without
> it?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/googlegsa/library/pull/13/files/60eaa81ab12e81f33dc645a55bfb29c87202c30a#r76381813>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIgXe6Rb00-whv7v9rZ2_1r-DeGXQPcaks5qjqKDgaJpZM4JtkW0>
> .
>